integrations/amazon_bedrock/tests/test_chat_generator_utils.py

```diff
@@ -985,152 +984,124 @@ def test_convert_streaming_chunks_to_chat_message_tool_call_with_empty_arguments
                 content=' designed to print "',
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.527426+00:00",
                 },
+                index=0,
             ),
             StreamingChunk(
                 content='Hello World" an',
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.590629+00:00",
                 },
+                index=0,
             ),
             StreamingChunk(
                 content="d doesn't require any parameters",
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.682261+00:00",
                 },
+                index=0,
             ),
             StreamingChunk(
                 content=". Let",
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.790526+00:00",
                 },
+                index=0,
             ),
             StreamingChunk(
                 content="'s go ahead an",
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.845332+00:00",
                 },
+                index=0,
             ),
             StreamingChunk(
                 content="d use",
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.990588+00:00",
                 },
+                index=0,
             ),
             StreamingChunk(
                 content=" it.",
                 meta={
                     "model": "anthropic.claude-3-5-sonnet-20240620-v1:0",
-                    "index": 0,
-                    "tool_calls": None,
-                    "finish_reason": None,
                     "received_at": "2025-07-31T08:46:07.994309+00:00",
                 },
+                index=0,
             ),
+            # TODO Does it make sense that this chunk is present?
```

Comment:
Could you clarify this comment?
Is this an "empty" chunk that we keep in order to have a 1:1 mapping with chunks emitted by Bedrock?